docs/serving/debugging-application-issues.md

```diff
@@ -43,46 +43,48 @@ kubectl get route <route-name> --output yaml
 
 The `conditions` in `status` provide the reason if there is any failure. For
 details, see Knative
-[Error Conditions and Reporting](https://github.com/knative/serving/blob/master/docs/spec/errors.md)(currently
+[Error Conditions and Reporting](https://github.com/knative/docs/blob/master/docs/serving/spec/knative-api-specification-1.0.md#error-signalling)(currently
 some of them are not implemented yet).
```

Comment:
"(currently some of them are not implemented yet)"

What does this mean (which ones)? The spec does not elaborate so it's unclear what is being referenced. Can this statement just be removed?

Reply:
I think the `severity` in error conditions is not been implemented.

Reply:
I read the `error-signalling` section and the relevant code. I am not find which one is not implemented. May be this could be removed.